rotkehlchen/tests/unit/decoders/test_thegraph.py

```diff
@@ -93,6 +98,155 @@ def test_thegraph_delegate(database, ethereum_inquirer):
     assert expected_events == events
 
 
+@pytest.mark.parametrize('ethereum_accounts', [[ADDY_ROTKI]])
+def test_thegraph_contract_deposit_gas(database, ethereum_inquirer):
+    tx_hash = deserialize_evm_tx_hash('0xf254ac1bbfbf07ca21042edd3ff78dad7c3158c8218598b5359b6e415e0977b7')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1706311139000)
+    gas, deposit_amount = '0.000626151499903872', '0.001135647343563552'
+    indexer = string_to_evm_address('0x7D91717579885BfCFec3Cb4B4C4fe71c1EedD4dE')
+    expected_events = [
+        EvmEvent(
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas)),
+            location_label=ADDY_ROTKI,
+            notes=f'Burned {gas} ETH for gas',
+            tx_hash=tx_hash,
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(deposit_amount), usd_value=FVal(0)),
```

Comment:
no need to set usd value. Default is zero

```suggestion
            balance=Balance(amount=FVal(deposit_amount)),
```